fem/bilininteg.hpp

```diff
@@ -2519,14 +2519,22 @@ class DivDivIntegrator: public BilinearFormIntegrator
 
 /** Integrator for
       (Q grad u, grad v) = sum_i (Q grad u_i, grad v_i) e_i e_i^T
-    for FE spaces defined by 'dim' copies of a scalar FE space. Where e_i
-    is the unit vector in the i-th direction.  The resulting local element
-    matrix is a block-diagonal matrix consisting of 'dim' copies of a scalar
-    diffusion matrix in each diagonal block. */
+    for vector FE spaces. Where e_i is the unit vector in the i-th direction.
+    The resulting local element matrix is square with dimension
+    <tt> vdim * dof </tt> where \c vdim is the vector dimension space and \c dof
+    is the local degrees of freedom. The integrator is not aware of the true
+    vector dimension and must use the \c VectorCoefficient,
+    \c MatrixCoefficient, or a caller-specified value to determine the vector
```

Comment:
Here you say the integrator may inherit the block size from the coefficients, but in practice it does not

Reply:
I think it does. `vdim` is set by calling `VectorCoefficient::GetVDim()` or `MatrixCoefficient::GetVDim()` in the constructor as appropriate.